src/instructlab/sdg/generate_data.py

```diff
@@ -400,6 +410,8 @@ def generate_data(
 
         if is_knowledge:
             # generate mmlubench data for the current leaf node
+            # Make sure checkpointing is disabled (we don't want this pipeline to load checkpoints from the main pipeline)
+            ctx.checkpoint_dir = None
```

Comment:
Nasty :)

How about, when creating `mmlu_pipe`:

```
# Make sure checkpointing is disabled (we don't want this pipeline to load checkpoints from the main pipeline)
mmlu_ctx = dataclasses.replace((ctx, checkpoint_dir=None)
mmlu_bench_pipe = mmlubench_pipe_init(mmlu_ctx)
```

Reply:
will do 

> Nasty :)
Yup, I knew when I was doing it